### PR TITLE
Linter fixes for poretools

### PR DESCRIPTION
**Pull request by bernt-matthias**
FOR CONTRIBUTOR:
* [ ] I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] License permits unrestricted use (educational + commercial)
* [ ] This PR adds a new tool or tool collection
* [ ] This PR updates an existing tool or tool collection
* [ ] This PR does something else (explain below)

### CODE REVIEW

File: tools/poretools/poretools_events.xml [hunk 1]
```diff
@@ -1,10 +1,9 @@
-<?xml version="1.0"?>
 <tool id="poretools_events" name="Extract nanopore events" version="@VERSION@.1">
```
Comment (bgruening) [inline tools/poretools/poretools_events.xml:1]: can we add profiles to all tools?

**Comment by bgruening** [comment]
Do we stop the CI on merge?

File: tools/poretools/macros.xml [hunk 2]
```diff
@@ -49,8 +50,8 @@
         <param name="input" ftype="fast5.tar" value="poretools-in1.tar.bz2" />
     </xml>
     <xml name="length_options">
-        <param name="min_length" argument="--min-length" type="integer" value="0" label="Minimum read length" />
-        <param name="max_length" argument="--max-length" type="integer" value="1000000000" label="Maximum read length" />
+        <param argument="--min-length" type="integer" value="0" label="Minimum read length" />
+        <param argument="--max-length" type="integer" value="1000000000" label="Maximum read length" />
     </xml>
     <token name="@MPLBACKEND@"><![CDATA[export MPLBACKEND="agg" &&]]></token>
     <token name="@profile@">18.09</token>
```
Comment (bernt-matthias) [inline tools/poretools/macros.xml:57]:
```suggestion
```

**Comment by bernt-matthias** [comment]
> Do we stop the CI on merge?

No, too many reasons to deploy: 

- changed (fixed) parameters
- changed profile

**Pull request merged**